frame/treasury/src/lib.rs

```diff
@@ -64,10 +64,13 @@ mod benchmarking;
 pub mod weights;
 
 use sp_std::prelude::*;
-use frame_support::{decl_module, decl_storage, decl_event, ensure, print, decl_error, PalletId};
+use frame_support::{
+	decl_module, decl_storage, decl_event, ensure, print, decl_error,
+	PalletId, BoundedVec, bounded_vec::TryAppendValue,
```

Comment:
```suggestion
	PalletId, BoundedVec,
```

Hmm are you sure we need this? We implemented it directly for `storage::types` so we won't need to import again.

Reply:
did get a compiler issue. let me see

Reply:
```
error[E0599]: no function or associated item named `try_append` found for struct `Approvals<T, I>` in the current scope
   --> frame/treasury/src/lib.rs:329:23
    |
178 | decl_storage! {
    | ------------- function or associated item `try_append` not found for this
...
329 |             Approvals::<T, I>::try_append(proposal_id).map_err(|_| Error::<T, I>::TooManyApprovals)?;
    |                                ^^^^^^^^^^ function or associated item not found in `Approvals<T, I>`
    |
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
    |
66  | use crate::sp_api_hidden_includes_decl_storage::hidden_include::bounded_vec::TryAppendValue;
```

If you know a better fix, happy to accept a follow up pr